UnitystationLauncher/Models/Server.cs

```diff
@@ -34,6 +34,8 @@ public string DownloadUrl
 
         public string InstallationName => ForkName + BuildVersion;
 
+        public string? Description => "BuildVersion: " + BuildVersion + " - Map: " + CurrentMap + " - Gamemode: " + GameMode + " - Time: " + IngameTime;
```

Comment:
Use string-interpolation:
```suggestion
        public string? Description => $"BuildVersion: {BuildVersion} - Map: {CurrentMap} - Gamemode: {GameMode} - Time: {IngameTime}";
```